src/components/App.tsx

```diff
@@ -421,6 +422,19 @@ export class App extends React.Component<AppProps, AppState> {
       notifyArcAboutFork(fiddle);
     }
   }
+
+  async forkIfNeeded(): Promise<boolean> {
```

Comment:
add a test?

Reply:
I don't see good way to test it without running whole thing in browser. 
Gonna track this here cause initial setup gonna be involved:
https://github.com/nearprotocol/NEARStudio/issues/121

Reply:
ok